terraform-aws-dataworks-pdm.tf

```diff
@@ -0,0 +1,69 @@
+resource "github_repository" "aws-dataworks-pdm" {
+  name             = "aws-dataworks-pdm"
+  description      = "This repo holds the Physical Data Model (PDM) and its tests"
+  auto_init        = true
+
+  allow_merge_commit     = false
+  delete_branch_on_merge = true
+  has_issues             = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  template {
+    owner = "${var.github_organization}"
+    repository = "dataworks-repo-template-terraform"
```

Comment:
If this is not a Terraform repo, you should use the `dataworks-repo-template` instead.